src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsGptPageParamOasis.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.driverprovider.UseUnstablePageLoadStrategy;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEng
+ */
+public class TestAdsGptPageParamOasis extends TemplateNoFirstLoad {
```

Comment:
https://github.com/Wikia/selenium-tests/blob/aden-2400/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsGptPageParamOasis.java#L23

As you can see this method is alone here.